### DISCUSSION TITLE
rclone for raspberry pi sync with dropbox cloud

### DISCUSSION THREAD

**Discussion by token386** [Geektoolkit/Dynaframe3#112] [Ideas]
I have installed rclone for raspberry (arm: https://rclone.org/downloads/), I have managed to synchronize the folder of images with dropbox perfectly. Good for the show. great job.

(https://pimylifeup.com/raspberry-pi-rclone/)

I have a script marked in crontab every 5 minutes that launches rclone by syncing /home/pi/Pictures with a folder in dropbox.

**Comment by dturn06** [comment]
Hello, I have rclone set as a cron job at reboot, but the issue that I am running into is that Dynaframe is trying to run before the cron job, and my images folder is not yet loaded into the system. This is causing the program to end as there are no pictures. A workaround is just to have one image in a different playlist on the local device, but I was wondering if you had run into this, and if so, how you have switched which program runs first.

**Reply by Geektoolkit** [reply]
I honestly use sleeps since I don't reboot often. A 30 second sleep may work to force Rclone to go after. In later builds it'll show a "no images found" page and then refresh the playlist in the interval. I think 2.19 may have that. 
  I'm working hard on the Dynaframe pro release which will have a lot of fixes for stuff like this

**Comment by qwksilver** [comment]
This may actually go away in the stable version. 2.18 I think.  2.23 may also have less issues with this, with 2.24 after rclone downloads the images folder, it will resync on bootup.  cron jobs are timer based so you could set one up for a sudo reboot at 2am to make sure everything is reindexed for the new day.  i turn all my equipment off and on constantly so I have less issues with database maint.
